.travis.yml

```diff
@@ -1,30 +1,45 @@
-sudo: false
 language: node_js
+
+os:
+  - linux
+
 node_js:
   - "10"
   - "12"
   - "14"
 
-os:
-  - linux
-
 script:
   - npm run postinstall
   - npm run build
   - npm run test:ci
 
 env: TASK=test
+
+cache:
+  yarn: true
+
+before_install:
+  - sudo apt-key adv --fetch-keys http://dl.yarnpkg.com/debian/pubkey.gpg
+  - echo "deb http://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+  - sudo apt-get update -qq
+  - sudo apt-get install -y -qq yarn
```

Comment:
Is there a better way to install `yarn` that does not require `sudo`?

Reply:
Is `npm i -g yarn` safe?

Reply:
It should be safe, I think it's a good idea

Reply:
works